security-setup

```diff
@@ -95,6 +98,10 @@ class Component(object):
         security = self.read_security()
         yield security
         security['security_enabled'] = True
+        if self.args.no_ssl:
+            security.setdefault('enable_ssl', False)
+        else:
+            security.setdefault('enable_ssl', True)
```

Comment:
`security.setdefault('enable_ssl', not self.args.no_ssl)`.

Aside from that, why is this only a default? If someone's passing in this flag, they're explicitly saying to change this setting. `setdefault` won't do that.